addons/mail/models/mail_thread.py

```diff
@@ -1951,12 +1951,12 @@ def message_post(self, *,
                      body='', subject=None, message_type='notification',
                      email_from=None, author_id=None, parent_id=False,
                      subtype_xmlid=None, subtype_id=False, partner_ids=None,
-                     attachments=None, attachment_ids=None,
+                     attachments=None, attachment_ids=None, body_html=False,
```

Comment:
"body_is_html" ? "content_is_html" ? Body_html conflicts with several 'body_html' fields that exist.